bottlerocket/types/src/agent_config.rs

```diff
@@ -146,7 +146,7 @@ pub struct Ec2Config {
     /// The type of instance to spin up. m5.large is recommended for x86_64 and m6g.large is
     /// recommended for arm64 on eks. c3.large is recommended for ecs. If no value is provided
     /// the recommended type will be used.
-    pub instance_type: Option<String>,
+    pub instance_types: Vec<String>,
```

Comment:
Could/should we except both ways to avoid breaking changes?
@webern @etungsten

Reply:
I wouldn't want to have two fields (`instance_type` and `instance_types`). It might be possible with something like:

`Either<Option<String>, Option<Vec<String>>`

But I think we might be trying to hard for backwards compatibility for a 0.0.2 versioned system.

Reply:
Just to clarify: do the "thumbs up" mean that people are in favor of the `Either` option or that we shouldn't worry about backwards compatibility too much and can leave it as-is?

Reply:
Although unfortunate, I think what you have right now is fine.